smarts/core/__init__.py

```diff
@@ -25,9 +25,14 @@
 
 import random
 import uuid
+from functools import lru_cache, partial
+from pathlib import Path
 
 import numpy as np
 
+import smarts
```

Comment:
This caused an odd bug. It could not resolve the difference between `smarts` and `smarts.core.smarts`. I believe should change `smarts.core.smarts` to `smarts.core.engine`.